profiles/Rishav1707.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://avatars.githubusercontent.com/u/97666287?v=4",
```

Should be the provided image in the issue you created.

```suggestion
  "image": "https://user-images.githubusercontent.com/97666287/211783632-12ef84c0-3f6d-4bf8-850c-234981a2e458.png",
```